etc/eth-contracts/contracts/EvmErc20.sol

```diff
@@ -13,16 +13,37 @@ import "./IExit.sol";
  * `ERC20` functions.
  */
 contract EvmErc20 is ERC20, AdminControlled, IExit {
+    string private _name;
```

Comment:
I think we don't need this here. It is already in [ERC20](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol#L40)

Reply:
This field is `private` there